app/src/main/java/org/mozilla/fenix/autofill/AutofillConfirmActivity.kt

```diff
@@ -16,4 +17,9 @@ import org.mozilla.fenix.ext.components
 @RequiresApi(Build.VERSION_CODES.O)
 class AutofillConfirmActivity : AbstractAutofillConfirmActivity() {
     override val configuration: AutofillConfiguration by lazy { components.autofillConfiguration }
+
+    override fun finish() {
+        super.finish()
+        components.analytics.metrics.track(Event.CreditCardAutofill)
```

Comment:
@eliserichards We don't use this to autofill credit cards I believe. Can you confirm by testing this?